src/openloco/localisation/stringmgr.cpp

```diff
@@ -1,23 +1,407 @@
+                            uint16_t value = *(uint16_t*) args;
+                            args = (uint16_t*) args + 1;
+                            buffer = format_comma(value, buffer);
+                            break;
+                        }
+
+                        case 123 + 5:
+                        {
+                            uint16_t value = *(uint16_t*) args;
+                            args = (uint16_t*) args + 1;
+                            buffer = format_int((uint32_t) value, buffer);
+                            break;
+                        }
+
+                        case 123 + 6:
+                        {
+                            // sub_495B66
+                            break;
+                        }
+
+                        case 123 + 7:
+                        {
+                            // sub_495B5B
+                            break;
+                        }
+
+                        case 123 + 8:
+                        {
+                            uint16_t value = *(uint16_t*) args;
+                            args = (uint16_t*) args + 1;
+                            const char* sourceStr_ = sourceStr;
+                            buffer = format_string(buffer, value, args);
+                            sourceStr = sourceStr_;
+                            break;
+                        }
+
+                        case 123 + 9:
+                        {
+                            id = *(uint16_t*) sourceStr;
+                            sourceStr += 2;
+                            const char* sourceStr_ = sourceStr;
+                            buffer = format_string(buffer, id, args);
+                            sourceStr = sourceStr_;
+                            break;
+                        }
+
+                        case 123 + 10:
+                        {
+                            const char* sourceStr_ = sourceStr;
+                            sourceStr = (char*) args;
+                            args = (uint32_t*) args + 1;
+
+                            do
+                            {
+                                *buffer++ = *sourceStr++;
+                            }
+                            while (*sourceStr != '\0');
+
+                            buffer--;
+                            sourceStr = sourceStr_;
+                            break;
+                        }
+
+                        case 123 + 11:
+                            // format by string?
+                            break;
+
+                        case 123 + 12:
+                            // velocity
+                            break;
+
+                        case 123 + 13:
+                            // pop16
+                            args = (uint16_t*) args + 1;
+                            break;
+
+                        case 123 + 14:
+                            // push16
+                            args = (uint16_t*) args + 1;
```

Comment:
Shouldn't either pop or push be `- 1`?

Reply:
Er, you're right. `push` should be `- 1`, of course.